daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -1451,6 +1491,10 @@ case class UnparserTestCase(ptc: NodeSeq, parentArg: DFDLTestSuite)
       if (!isCrossTest(implString.get) ||
         parent.shouldDoWarningComparisonOnCrossTests)
         VerifyTestCase.verifyAllDiagnosticsFound(actual.getDiagnostics, optWarnings, implString)
+      
+      if (!isCrossTest(implString.get) ||
+        parent.shouldDoLogComparisonOnCrossTests)
```

Comment:
Why are there all these missing codecov lines? Seems like we have tests that should hit them?

Reply:
Much of the features for cross testing don't get exercised unless you run the IBM cross testing rig. 

To test these without that we'd have to build a mock cross tester that actually just uses daffodil.

Reply:
Ah right, I missed that this is only cross testing related. Not worth it to create a mock cross tester. Resolving this conversation.